docs/migration/4_0.md

```diff
@@ -1,3 +1,35 @@
 # Technical Changes in Spartacus 4.0
 
 ## Breaking Changes Introduced in 4.0
+
+## Automated Migrations for Version 4.0
+
+- `ProductVariantsModule` was removed from @spartacus/storefront. Use `@spartacus/product/variants` feature-library instead.
+- `ProductVariantsComponent` was removed from @spartacus/storefront. Use `ProductVariantsContainerComponent` from `@spartacus/product/variants/components` as a replacement.
+- `ProductVariantsModule` was removed from @spartacus/storefront. Use `ProductVariantsContainerModule` from `@spartacus/product/variants/components` as a replacement.
```

Comment:
I think new version of this module is actually: `ProductVariantsComponentsModule`

Reply:
No, we tidy it up a bit because previous approach/config was messy. Container component haven't had it's own module, which changed with migrating to feature-lib.